From b5d178a46e55c72ea160cdc4549efd120fd850ae Mon Sep 17 00:00:00 2001
Message-Id: <b5d178a46e55c72ea160cdc4549efd120fd850ae.1434478027.git.Jim.Somerville@windriver.com>
From: Jim Somerville <Jim.Somerville@windriver.com>
Date: Tue, 16 Jun 2015 14:00:29 -0400
Subject: [PATCH 1/1] Set pkgconfigdir to use the proper libdir

The hardcoding of "lib" in the path caused the wrong location to be
used, resulting in package config files not making it into the
target sysroot, and bitbake generating errors about files being
installed but not shipped.

Signed-off-by: Jim Somerville <Jim.Somerville@windriver.com>
---
 nih-dbus/Makefile.am | 2 +-
 nih/Makefile.am      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/nih-dbus/Makefile.am b/nih-dbus/Makefile.am
index 1c90adc..cbc1bb4 100644
--- a/nih-dbus/Makefile.am
+++ b/nih-dbus/Makefile.am
@@ -49,7 +49,7 @@ nihdbusinclude_HEADERS = \
 	test_dbus.h
 
 
-pkgconfigdir = $(prefix)/lib/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libnih-dbus.pc
 
 
diff --git a/nih/Makefile.am b/nih/Makefile.am
index 2a8175d..3234962 100644
--- a/nih/Makefile.am
+++ b/nih/Makefile.am
@@ -71,7 +71,7 @@ nihinclude_HEADERS = \
 	test_hash.h
 
 
-pkgconfigdir = $(prefix)/lib/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libnih.pc
 
 
-- 
1.8.3.2

